sdk/storage/storage-blob/rollup.base.config.js

```diff
@@ -94,7 +94,10 @@ export function browserConfig(test = false, production = false) {
       banner: banner,
       format: "umd",
       name: "azblob",
-      sourcemap: true
+      sourcemap: true,
+      globals: {
+        "fs-extra": "fs-extra"
```

Comment:
Wait! fs-extra was imported in the recorder.ts file, which is causing this.
recorder.ts can be deleted since we are using the unified common recorder package. I'll make a PR.